Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -131,6 +133,25 @@ export class JavaFilesExtractor {
         }    
     }
 
+    /**
+     * Creates a list of directories on the root level of structure.
+     * @param pathsArray - contains paths to all the files inside the structure
+     * @param root - path to the directory we want to get the structure of
+     */
+
+    private static sliceStructure(pathsArray: Array<string>, root: string = pathsArray[0]): Array<string>{
+        const dirPathLength = root.length;
+        const structureObject = {};
```

Comment:
Could you add type to this dictionary? It's unclear from init value here which type key/value have